app/Http/Controllers/API/WpOrg/Themes/ThemeController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Controllers\API\WpOrg\Themes;
+
+use App\Http\Controllers\Controller;
+use App\Models\ApiResultsResponse;
+
+class ThemeController extends Controller
+{
+    private $actions = [
+        'query_themes' => 'doQueryThemes',
+    ];
+
+    public function sendResponse($data, $statusCode = 200)
```

Comment:
Should not be public.  The only public methods on a controller should be the ones invoked by routes.

Also, it would be better if the request were passed in to this method rather than the request() function used, but TBH as long as it's confined to the controller I don't care that much.